packages/client/hmi-client/src/services/models/simulation-service.ts

```diff
@@ -302,3 +326,56 @@ export async function simulationPollAction(
 		error: true
 	};
 }
+
+type MessageHandler = (message: any) => void;
+
+export class EventSourceManager {
```

Comment:
Feels like this class deserves its own file in `/utils` folder?

Reply:
agreed - this has been done